packages/core/test/pseudo-states.spec.ts

```diff
@@ -626,6 +649,64 @@ describe('pseudo-states', () => {
                 });
             });
 
+            it('should warn when pseudo-class expects params but none was given', () => {
```

Comment:
all these test are also related to the fact that there is no default value. Add this into the test title

Reply:
@barak007 done